docs/how_to/pytorch_install/pytorch_install.md

```diff
@@ -1,154 +1,152 @@
+table, choose ROCm from the _Compute Platform_ row.
 
-```{figure} ../../data/how_to/magma_install/image.006.png
-:name: Installation-Matrix-from-Pytorch
----
-align: center
----
-Installation Matrix from Pytorch
-```
+1. Choose one of the following three options:
 
-To install PyTorch using the wheels package, follow these installation steps:
+   **Option 1:**
 
-1. Choose one of the following options:
-   a. Obtain a base Docker image with the correct user-space ROCm version
-      installed from
-      [https://hub.docker.com/repository/docker/rocm/dev-ubuntu-20.04](https://hub.docker.com/repository/docker/rocm/dev-ubuntu-20.04).
+   Download a base Docker image with the correct user-space ROCm version.
+   | Base OS         | Docker image                   | Link to Docker image|
+   |----------------|-----------------------------|----------------|
+   | Ubuntu 20.04 | `rocm/dev-ubuntu-20.04` | [https://hub.docker.com/r/rocm/dev-ubuntu-20.04](https://hub.docker.com/r/rocm/dev-ubuntu-20.04)
+   | Ubuntu 22.04 | `rocm/dev-ubuntu-22.04` | [https://hub.docker.com/r/rocm/dev-ubuntu-22.04](https://hub.docker.com/r/rocm/dev-ubuntu-22.04)
+   | CentOS 7        | `rocm/dev-centos-7` | [https://hub.docker.com/r/rocm/dev-centos-7](https://hub.docker.com/r/rocm/dev-centos-7)
 
-   or
+   **Option 2:**
 
-   b. Download a base OS Docker image and install ROCm following the
-      installation directions in the section
-      [Installation](../../deploy/linux/install.md). ROCm 5.2 is installed in
-      this example, as supported by the installation matrix from
-      <https://pytorch.org/>.
+   Download a base OS Docker image and install ROCm using the directions in the
+   [Installation](../../deploy/linux/install.md) section.
 
-   or
+   **Option 3:**
 
-   c. Install on bare metal. Skip to Step 3.
+   Install on bare metal. If using this method, skip step 2 (starting a Docker container).
 
-      ```bash
-      docker run -it --device=/dev/kfd --device=/dev/dri --group-add video rocm/dev-ubuntu-20.04:latest
-      ```
+   ```bash
+   docker run -it --device=/dev/kfd --device=/dev/dri --group-add video rocm/dev-ubuntu-20.04:latest
+   ```
 
-2. Start the Docker container, if not installing on bare metal.
+2. Start the Docker container (skip this step for bare metal installations).
 
    ```dockerfile
    docker run -it --device=/dev/kfd --device=/dev/dri --group-add video rocm/dev-ubuntu-20.04:latest
    ```
 
-3. Install any dependencies needed for installing the wheels package.
+3. Install the required dependencies for the wheels package.
 
    ```bash
    sudo apt update
-   sudo apt install libjpeg-dev python3-dev
+   sudo apt install libjpeg-dev python3-dev python3-pip
    pip3 install wheel setuptools
    ```
 
-4. Install torch, `torchvision`, and `torchaudio` as specified by the installation
-   matrix.
+### Using MIOpen kdb files with ROCm PyTorch wheels
```

Comment:
@LisaDelaney wrong merge here
from `### Using MIOpen kdb files with ROCm PyTorch wheels`
to `3. ./install_kdb_files_for_pytorch_wheels.sh`
It looks like an optional step after full installation of Pytorch. You can move it down behind `pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/rocm5.6/` as step 5. 
@jithunnair-amd check me please

Text for this place:
4. Install `torch`, `torchvision`, and `torchaudio` as specified by the installation
   matrix.

   :::{note}
   The following command uses the ROCm 5.6 PyTorch wheel. If you want a different version of ROCm,
   modify the command accordingly.
   :::

```bash
   pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/rocm5.6/
```

@LisaDelaney  and after all you can put 
5. Using MIOpen kdb files with ROCm PyTorch wheels (optional) 
PyTorch uses [MIOpen](https://github.com/ROCmSoftwarePlatform/MIOpen) for machine learning
primitives, which are compiled into kernels at runtime. Runtime compilation causes a small warm-up
phase when starting PyTorch, and MIOpen kdb files contain precompiled kernels that can speed up
application warm-up phases. For more information, refer to the
{doc}`MIOpen installation page <miopen:install>`.

MIOpen kdb files can be used with ROCm PyTorch wheels. However, the kdb files need to be placed in
a specific location with respect to the PyTorch installation path. A helper script simplifies this task by
taking the ROCm version and GPU architecture as inputs. This works for Ubuntu and CentOS.

You can download the helper script here:
[install_kdb_files_for_pytorch_wheels.sh](https://raw.githubusercontent.com/wiki/ROCmSoftwarePlatform/pytorch/files/install_kdb_files_for_pytorch_wheels.sh)

After installing ROCm PyTorch wheels, run the following code:

1. (Optional) `export GFX_ARCH=gfx90a`
2. (Optional) `export ROCM_VERSION=5.5`
3. `./install_kdb_files_for_pytorch_wheels.sh`

Reply:
@dnikolaev-amd mind having a look to see if I addressed everything appropriately (I'm a little unsure about this comment in particular)?